src/main/kotlin/world/anhgelus/msmp/alteredrespawn/AlteredRespawn.kt

```diff
@@ -1,16 +1,20 @@
 package world.anhgelus.msmp.alteredrespawn
 
+import org.bukkit.Bukkit
+import world.anhgelus.msmp.alteredrespawn.listener.RegisterListener
+import world.anhgelus.msmp.alteredrespawn.listener.dead
 import world.anhgelus.msmp.msmpcore.PluginBase
 
 class AlteredRespawn: PluginBase() {
     override val pluginName: String = "AlteredRespawn"
 
-    override fun disable() {
-
-    }
-
     override fun enable() {
+        Bukkit.getPluginManager().registerEvents(RegisterListener, this);
+        Bukkit.getLogger().info("Plugin initialisé")
```

Comment:
You must do this:

```kotlin
INSTANCE = this
LOGGER = logger
````

The `Bukkit.getLogger().info("Plugin initialisé")` is useless because it is already did automatically and you must use the English.